src/events/crawler/scrapers/USA/AR/index.js

```diff
@@ -0,0 +1,48 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  state: 'AR',
+  country: 'USA',
+  url:
+    'https://services.arcgis.com/PwY9ZuZRDiI5nXUB/ArcGIS/rest/services/ADH_COVID19_Positive_Test_Results/FeatureServer/0/query?f=json&where=1%3D1&returnGeometry=false&spatialRel=esriSpatialRelIntersects&outFields=*',
+  type: 'json',
+  sources: [
+    {
+      name: 'Arkanas Department of Health',
+      url: 'https://www.healthy.arkansas.gov/programs-services/topics/novel-coronavirus',
+      description: ''
+    }
+  ],
```

Comment:
Care to add yourself as `maintainer: [{...}]` for these?

Reply:
Will do!